met-web/src/components/engagement/view/ActionContext.tsx

```diff
@@ -43,48 +45,54 @@ export const ActionContext = createContext<EngagementViewContext>({
     updateMockStatus: (status: SubmissionStatus) => {
         /* nothing returned */
     },
+    subscriptionText: '',
 });
 
 export const ActionProvider = ({ children }: { children: JSX.Element | JSX.Element[] }) => {
-    const { engagementId, token } = useParams<EngagementParams>();
+    const { engagementId, subscriptionStatus, scriptionKey } = useParams<EngagementParams>();
     const navigate = useNavigate();
     const dispatch = useAppDispatch();
     const [savedEngagement, setSavedEngagement] = useState<Engagement>(createDefaultEngagement());
     const [mockStatus, setMockStatus] = useState(savedEngagement.submission_status);
     const [widgets, setWidgets] = useState<Widget[]>([]);
     const [isEngagementLoading, setEngagementLoading] = useState(true);
     const [isWidgetsLoading, setIsWidgetsLoading] = useState(true);
+    const [subscriptionText, setSubscriptionText] = useState('');
 
     const [getWidgetsTrigger] = useLazyGetWidgetsQuery();
 
     useEffect(() => {
-        verifySubscribeToken();
-    }, [token]);
+        verifySubscribeKey();
+    }, [scriptionKey]);
 
     useEffect(() => {
         setMockStatus(savedEngagement.submission_status);
     }, [savedEngagement]);
 
-    const verifySubscribeToken = async () => {
+    const verifySubscribeKey = async () => {
         try {
-            if (!token) {
+            if (!scriptionKey) {
                 return;
             }
-            await verifyEmailVerification(token);
-            dispatch(
-                openNotificationModal({
-                    open: true,
-                    data: {
-                        header: 'Subscribed Successfully',
-                        subText: [
-                            {
-                                text: 'Your email has been verified. You will now receive news and updates from us.',
-                            },
-                        ],
-                    },
-                    type: 'update',
-                }),
-            );
+            if (subscriptionStatus == 'subscribe') {
```

Comment:
we should store this 'subscribe' in constant file possible under same Folder, SUBSCRIBE = 'subscribe' same for the other

Reply:
added in a constant file